.github/workflows/inspektor-gadget.yml

```diff
@@ -691,25 +691,6 @@ jobs:
             echo "aks=false" >> $GITHUB_OUTPUT
           fi
 
-  check-readme-up-to-date:
-    name: Check that README is up to date
-    # level: 1
-    needs: build-kubectl-gadget
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v3
-    - name: Get kubectl-gadget-linux-amd64.tar.gz from artifact.
-      uses: actions/download-artifact@v3
-      with:
-        name: kubectl-gadget-linux-amd64-tar-gz
-        path: /home/runner/work/inspektor-gadget/
-    - name: Check that README is up to date
-      shell: bash
-      run: |
-        tar zxvf /home/runner/work/inspektor-gadget/kubectl-gadget-linux-amd64.tar.gz
-
-        perl tools/check-readme.pl ./kubectl-gadget README.md
-
```

Comment:
If this is causing issues we can remove it and add in a follow up PR.

Reply:
Mmmm maybe it's a silly question but, what about running `./kubectl-gadget update-catalog` before the check?

Reply:
> Mmmm maybe it's a silly question but, what about running `./kubectl-gadget update-catalog` before the check?

At the original location in the workflow? There's no cluster running where it could get the catalog from in that test. After running the integration tests that's the case, though (and then no update is needed).

Reply:
I just suggest adding a comment to say that the conditional on default and docker is just to run it once, not because such a check makes only sense for that combination.